packages/yew-bootstrap/bootstrap-icons-v1.11.3/bootstrap-icons.css

```diff
@@ -1,14 +1,14 @@
 /*!
- * Bootstrap Icons v1.10.5 (https://icons.getbootstrap.com/)
- * Copyright 2019-2023 The Bootstrap Authors
+ * Bootstrap Icons v1.11.3 (https://icons.getbootstrap.com/)
```

Comment:
The version is identified here, so no need to identify it in the directory and complicate things.